app/services/preserved_object_handler.rb

```diff
@@ -133,18 +133,25 @@ def update_version
 
   private
 
-  def create_db_objects(status)
+  def create_db_objects(status, validate=false)
```

Comment:
would it be more descriptive to say "validated" instead of "validate"?  It looks like it refers to whether or not validation has been performed already.

Reply:
agreed w/ @ndushay on verb tense